.sbtrc

```diff
@@ -0,0 +1,4 @@
+# Test run control
+alias test=; testOnly -- -l IntegrationTest
+alias testIT=; testOnly -- -t IntegrationTest
```

Comment:
I liked the `testDB` alias 😉

Reply:
```suggestion
alias testIT=; testOnly -- -t IntegrationTest
alias testDBl=; "module database" testOnly
```

Reply:
See the latest solution. `testDB` is there for database tests written using ScalaTest framework.

Reply:
In the latest `test` alias, there were found a problem:
- `test` command fails when db does not exist - initialized.

1. There is an unexpected run of the test method in DBTestSuite (balta)
2. The scala test still follows the exclude. When db is prepared, output from the test run is only the list of Suite without started tests - it looks as expected.